src/components/NavBar/NavBar.module.css

```diff
@@ -2,39 +2,58 @@
   display: flex;
   flex-direction: row;
   justify-content: space-between;
-  background-color: #262e4b;
+  background-color: #232627;
   width: 100%;
   z-index: 1000;
 }
 
 .navBarContainer {
   display: flex;
+  width: 100%;
+  justify-content: space-between;
+  padding: 0.8rem;
 }
 
 .navContainer {
   display: flex;
 }
 
 .navLink {
-  margin: auto 10px;
+  font-family: 'Montserrat';
```

Comment:
I havent used Montserrat anywhere in the figma, did you take a look at what fonts i used?